contrib/terraform/aws/terraform.tfvars.example

```diff
@@ -28,5 +27,6 @@ aws_cluster_ami = "ami-903df7ff"
 
 #Settings AWS ELB
 
-aws_elb_api_port = 443
-k8s_secure_api_port = 443
+aws_elb_api_port = 6443
+k8s_secure_api_port = 6443
+kube_insecure_apiserver_address = 0.0.0.0
```

Comment:
Unfortunately AWS ELB requires a HTTP 200 Return Code for the health check of the instances. Hence the API Server has to accept requests from other instances inside the AWS VPC. If I use the secure port (6443) the server returns authorization required which isnt a HTTP 200 code.